hummingbot/market/binance/binance_market.pyx

```diff
@@ -577,13 +578,20 @@ cdef class BinanceMarket(MarketBase):
                 event_type = event_message.get("e")
 
                 if event_type == "executionReport":
-                    client_order_id = event_message.get("c")
+                    execution_type = event_message.get("x")
+                    if execution_type != "CANCELED":
```

Comment:
maybe add a link to documentation for this since this seems weird

Reply:
done